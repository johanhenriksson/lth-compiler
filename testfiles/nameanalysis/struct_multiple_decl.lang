struct one {
    int two;
}

struct one {
    int three;
}
