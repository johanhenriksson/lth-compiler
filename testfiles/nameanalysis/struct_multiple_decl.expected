Error at line 5,1: Struct one cannot be declared more than once
